src/multi_vector_simulator/D2_model_constraints.py

```diff
@@ -57,6 +59,13 @@ def add_constraints(local_energy_system, dict_values, dict_model):
     -----
     The constraints can be validated by evaluating the LP file.
     Additionally, there are validation tests in `E4_verification_of_constraints`.
+
+    Tested with:
+    - D2.test_add_constraints_maximum_emissions()
+    - D2.test_add_constraints_maximum_emissions_None()
+    - D2.test_add_constraints_minimal_renewable_share() (tba)
+    - D2.test_add_constraints_minimal_renewable_share_None() (tba)
```

Comment:
Are they implemented? If they are not, I would rather have this as an issue then as (tba) in the code. Although we are meeting 100 issues now D=